mxcube3/routes/limsutils.py

```diff
@@ -178,6 +178,9 @@ def lims_existing_session(login_res):
 def lims_is_inhouse(login_res):
     return login_res.get("Session", {}).get("is_inhouse", False)
 
+def lims_is_staff(loginID):
```

Comment:
Hm, we need to double check the usage of `lims_is_staff` and `lims_is_inhouse` and especially if its compatible with whats done in ISPyBClient2, the lims_is_staff check is basically already done in:

https://github.com/mxcube/HardwareRepository/blob/3caa18f0d853167d43e00bead253917c20df9a56/HardwareObjects/ISPyBClient2.py#L775

The problem is that the inhouse proposals definition in session is used for both users and proposals. Actually the only thing that can be inhouse according to the old definition is a proposal (as it has to do with data storage). With many proposals per user this should be changed to in_house_proposals, we could then imagine to have a in_house_users with usernames.